./userDB.py:31:80: E501 line too long (94 > 79 characters)
